templates/shortcode-receipt.php

```diff
@@ -148,7 +148,7 @@
 						?>
 
 						<div class="edd_purchase_receipt_product_name">
-							<?php echo esc_html( $item['name'] ); ?>
+							<?php echo esc_html( edd_get_cart_item_name( $item ) ); ?>
 							<?php if( ! is_null( $price_id ) ) : ?>
 							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo edd_get_price_option_name( $item['id'], $price_id, $payment->ID ); ?></span>
```

Comment:
@PareshRadadiya The problem with that is it changes the markup of the checkout, which could break users styles. While it is nicer using a simple function, we can't do that for backwards compatibility's sake.